db/db_sst_test.cc

```diff
@@ -14,6 +14,7 @@
 #include "util/sst_file_manager_impl.h"
 
 namespace rocksdb {
+#ifndef ROCKSDB_LITE
```

Comment:
I think `ifndef ROCKSDB_LITE` should be added only around FlushedFileCollector and DBSSTTest .CancellingManualCompactionsWorks test. (introduced in #3670).

Reply:
well currently the `ifndef ROCKSDB_LITE` is covering line 125 to line 1026.
My change basically excludes two more tests `DontDeletePendingOutputs` and `SSTsWithLdbSuffixHandling` for rocksdb lite. 
As long as these two tests are not must-have for rocksdb lite, I think we are all good here?